src/vs/workbench/browser/parts/editor/media/tabstitlecontrol.css

```
@@ -411,16 +411,26 @@
 	height: 22px; /* tweak the icon size of the editor labels when icons are enabled */
 }
 
+.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-breadcrumbs .breadcrumbs-control .monaco-icon-label > .monaco-icon-label-container > .monaco-icon-name-container > .label-name {
+	font-weight: bold;
+}
+
 .monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-breadcrumbs .breadcrumbs-control .outline-element-icon {
 	padding-right: 3px;
 	height: 22px; /* tweak the icon size of the editor labels when icons are enabled */
 	line-height: 22px;
 }
 
+.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-breadcrumbs .monaco-breadcrumbs {
+	min-height: 28px; /* this overrides the element style height-property */
```

In order to adhere to the GNOME guidelines, let's make the breadcrumb background rounded and decrease its width. We should aim for a close-to-perfect alignment of around 902 pixels, although I haven't measured it precisely

```suggestion
	min-height: 28px; /* this overrides the element style height-property */
	border-radius: 6px;
  width: 902px !important;
```